Plugin/CppApi/source/ArcGISRuntimeToolkit.cpp

```diff
@@ -45,11 +58,36 @@ ArcGISRuntimeToolkit::ArcGISRuntimeToolkit(QObject* parent) :
 #endif
 }
 
+/*!
+  \brief Type registration function to ensure the types are accessible
+  in the QML environment.
+
+  You can use this method if you want to customize the import path. For ease
+  of use, \l registerToolkitTypes can be used instead without needing to instantiate
+  this class.
+
+  \list
+  \li \a uri - The namespace in which to register the types.
```

Comment:
we should provide an example of this. It's not clear to me what input we would expect. Is it "myProject::mySubNamespace" or "myProject.MyNamespace"?

Reply:
I agree. I looked for an example but could not find one. I will look some more.

Reply:
(that can be done later - not required for this)